### ISSUE TITLE
set_manual_preset_runtime is missing since last update

### ISSUE THREAD

**Issue by Pirol62** [sebr/bhyve-home-assistant#86]
**Describe the bug**

Since the last update to 2.0.9 the service bhyve.set_manual_preset_runtime is missing
**Expected behaviour**

I used the service in an automation
````
- service: bhyve.set_manual_preset_runtime
      data:
        entity_id: switch.zone_1_zone
        minutes: >
          {% if now().month in (1,2,11,12) %}
            15
          {% elif now().month in (3,4,9,10) %}
            20
          {% elif now().month in (6,7) %}
            30
          {% else %}
            25
          {% endif %}
````
**BHyve devices**

**Logs**
Logger: homeassistant.components.automation.set_irrigation_runtime
Source: components/automation/__init__.py:522
Integration: Automatisierung (documentation, issues)
First occurred: 1. November 2021, 00:05:20 (1 occurrences)
Last logged: 1. November 2021, 00:05:20

Error while executing automation automation.set_irrigation_runtime: Unable to find service bhyve.set_manual_preset_runtime

```log
DEBUG (MainThread) [custom_components.bhyve] Devices:
DEBUG (MainThread) [custom_components.bhyve] Programs:
```

**Comment by sebr** [comment]
Yes, I removed it as I don't think it ever worked?

Did it work for you? Can you confirm if you downgrade to the previous version please?

Which device do you have?

**Comment by Pirol62** [comment]
It worked perfectly. I have the Orbit 94990.

**Comment by sebr** [comment]
Released 2.0.10

**Comment by Pirol62** [comment]
Thanks mate. Everything works again.
